src/static/js/components/opinionModal/view.js

```diff
@@ -60,6 +67,15 @@ OpinionModalView.prototype.subscribers = function () {
   events.closeModal.subscribe(function() {
     events.closeOpinionModal.publish();
   });
+
+  events.showSubmissionCue.subscribe(function(opinion) {
+    self.showSubmissionCue(opinion);
+  });
+
+  events.showNextSuggestion.subscribe(function() {
+    self.nextOpinionElements.removeClass('-ripple -inactive');
```

Comment:
Should this commands be a function?

Reply:
I believe I fixed that. Please review once more.